internal/engine/experiment/signal/signal.go

```diff
@@ -15,7 +15,7 @@ import (
 
 const (
 	testName    = "signal"
-	testVersion = "0.2.0"
+	testVersion = "0.2.1"
```

Comment:
You should also update the corresponding test in `signal_test.go`, which fails like this:

```
signal_test.go:21: unexpected version
```

Reply:
Done!

Reply:
🙏